cluster-autoscaler/FAQ.md

```diff
@@ -175,6 +176,15 @@ For example for a node label of `foo=bar` you would tag the ASG with:
     "Key": "k8s.io/cluster-autoscaler/node-template/label/foo"
 }
 ```
+
+### How can I prevent Cluster Autoscaler from scaling down a particular node?
+
+From CA 0.7 node will be excluded from scale down if it has no scale down
+annotation:
+
+```
+"cluster-autoscaler.kubernetes.io/scale-down-disabled": "true"
+```
```

Comment:
Add example how to set the annotation with kubectl.

Reply:
Done